osxkeychain/osxkeychain_darwin.go

```diff
@@ -134,8 +134,21 @@ func (h Osxkeychain) List() (map[string]string, error) {
 	return resp, nil
 }
 
+// alwaysHTTPS takes a url and makes sure it always has the https protocol
+func alwaysHTTPS(url string) string {
```

Comment:
`^\w+://` instead I'd say so that you match one or more `word chars` (a-z, A-Z, 0-9, '_') before a `://`